### ISSUE TITLE
TimelineStep (disabled): insufficient color contrast

### ISSUE THREAD

**Issue by mjcarsjens** [nl-design-system/denhaag#412]
contrast between text and white background is too small for a disabled TimelineStep.

## Acceptance criteria
- [ ] Figma design should be updated to display disabled state that passes contrast requirements
- [ ] Component updated according to new design
